APITaxi2/tests/test_vehicles.py

```diff
@@ -145,12 +134,11 @@ def test_ok(self, operateur, admin, QueriesTracker):
         })
         assert resp.status_code == 201
         vehicle = Vehicle.query.options(
-            joinedload(Vehicle.descriptions).joinedload(VehicleDescription.model),
-            joinedload(Vehicle.descriptions).joinedload(VehicleDescription.constructor)
+            joinedload(Vehicle.descriptions)
         ).filter_by(licence_plate='licence2').one()
         assert vehicle.descriptions
-        assert vehicle.descriptions[0].model is None
-        assert vehicle.descriptions[0].constructor is None
+        assert not vehicle.descriptions[0].model
+        assert not vehicle.descriptions[0].constructor
```

Comment:
You want to test both `None` and `''` or purpose?

Reply:
The field is not nullable anymore, so it tests it is `''`